FirebaseTestProject/ViewModels/AuthViewModel.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  AuthViewModel.swift
+//  AuthViewModel
+//
+//  Created by Tim Miller on 8/12/21.
+//
+
+
+import FirebaseAuth
+import SwiftUI
+
+@MainActor class AuthViewModel: ObservableObject {
+  
+  private let auth = Auth.auth()
+  
+  @AppStorage("userSignedIn") var userSignedIn: Bool = false
+  
+  func getUser() -> String {
+    guard let user = auth.currentUser?.email else {
+      return "No User"
+    }
+    return user
+  }
+  
+  func signIn(email: String, password: String) async {
+    do {
+      try await auth.signIn(withEmail: email, password: password)
+      UserDefaults.standard.set(true, forKey: "userSignedIn")
```

Comment:
I think Firebase can do this automatically?  It might be the default behavior: https://www.google.com/search?q=firebase+auth+persistence&rlz=1C5GCEM_enUS965US965&oq=firebase+auth+persis&aqs=chrome.0.0i512j69i57j0i512l4j0i22i30l4.2733j0j7&sourceid=chrome&ie=UTF-8

Reply:
Just removed the user defaults code locally and it seemed to still persist the authentication.